Registration login problem

![Image](https://github.com/user-attachments/assets/06363ed6-b94f-4efd-9c3a-f4d46209b980)

The project is deployed on the server, and when using remote login, the address called is not the address of the server, but localhost
The following error is reported：Network error or server is unreachable
Is there a place for configuration？
thanks

additional information 
2025-02-10 01:20:43 2025-02-10 09:20:43+00:00 [Note] [Entrypoint]: Entrypoint script for MySQL Server 8.0.41-1.el9 started.
2025-02-10 01:20:44 2025-02-10 09:20:44+00:00 [Note] [Entrypoint]: Switching to dedicated user 'mysql'
2025-02-10 01:20:44 2025-02-10 09:20:44+00:00 [Note] [Entrypoint]: Entrypoint script for MySQL Server 8.0.41-1.el9 started.
2025-02-10 01:20:44 2025-02-10 09:20:44+00:00 [ERROR] [Entrypoint]: Database is uninitialized and password option is not specified
2025-02-10 01:20:44     You need to specify one of the following as an environment variable:
2025-02-10 01:20:44     - MYSQL_ROOT_PASSWORD
2025-02-10 01:20:44     - MYSQL_ALLOW_EMPTY_PASSWORD
2025-02-10 01:20:44     - MYSQL_RANDOM_ROOT_PASSWORD

--- a/backend/Dockerfile
+++ b/backend/Dockerfile
@@ -16,6 +16,10 @@ COPY requirements.txt .
 # Install Python packages
 RUN pip install --no-cache-dir -r requirements.txt
 
+# Copy entrypoint script first
+COPY entrypoint.sh .
+RUN chmod +x entrypoint.sh
+
 # Copy application files
 COPY . .
 

